### ISSUE TITLE
Locales not applied

### ISSUE THREAD

**Issue by bereldhuin** [amcharts/amcharts5#85]
Hi,

I set the root locale to french (fr_FR) but nothing changed on the graph : the number format is still in english format (1,000.00 instead of 1 000,00) and monthes are not translated in time series.

Here is an example : 

```
<style>
    #chartdiv {
        width: 100%;
        height: 500px;
    }
</style>

<!-- Resources -->
<script src="https://cdn.amcharts.com/lib/5/index.js"></script>
<script src="https://cdn.amcharts.com/lib/5/xy.js"></script>
<script src="https://cdn.amcharts.com/lib/5/themes/Animated.js"></script>
<script src="https://cdn.amcharts.com/lib/5/locales/fr_FR.js"></script>


<!-- HTML -->
<div id="chartdiv"></div>


<!-- Chart code -->
<script>
    //am5.ready(function() {

    // Create root element
    // https://www.amcharts.com/docs/v5/getting-started/#Root_element
    var root = am5.Root.new("chartdiv");

    // Set locale
    root.locale = am5locales_fr_FR;


    // Set themes
    // https://www.amcharts.com/docs/v5/concepts/themes/
    root.setThemes([
        am5themes_Animated.new(root)
    ]);


    // Create chart
    // https://www.amcharts.com/docs/v5/charts/xy-chart/
    var chart = root.container.children.push(am5xy.XYChart.new(root, {
        //panX: true,
        //panY: true,
        //wheelX: "panX",
        //wheelY: "zoomX"
    }));


    // Add cursor
    // https://www.amcharts.com/docs/v5/charts/xy-chart/cursor/
    var cursor = chart.set("cursor", am5xy.XYCursor.new(root, {
        //behavior: "none"
    }));
    cursor.lineY.set("visible", false);


    // The data
    var data = [{
        "dte": "2021-04-01",
        "cars": 1587.2,
        "motorcycles": 650,
        "bicycles": 121
    }, {
        "dte": "2021-04-02",
        "cars": 1567.3,
        "motorcycles": 683,
        "bicycles": 146
    }, {
        "dte": "2021-04-03",
        "cars": 1617,
        "motorcycles": 691,
        "bicycles": 138
    }, {
        "dte": "2021-04-04",
        "cars": 1630,
        "motorcycles": 642,
        "bicycles": 127
    }, {
        "dte": "2021-04-05",
        "cars": 1660,
        "motorcycles": 699,
        "bicycles": 105
    }, {
        "dte": "2021-04-06",
        "cars": 1683,
        "motorcycles": 721,
        "bicycles": 109
    }, {
        "dte": "2021-04-07",
        "cars": 1691,
        "motorcycles": 737,
        "bicycles": 112
    }, {
        "dte": "2021-04-08",
        "cars": 1298,
        "motorcycles": 680,
        "bicycles": 101
    }, {
        "dte": "2021-04-09",
        "cars": 1275,
        "motorcycles": 664,
        "bicycles": 97
    }, {
        "dte": "2021-04-10",
        "cars": 1246,
        "motorcycles": 648,
        "bicycles": 93
    }, {
        "dte": "2021-04-11",
        "cars": 1318,
        "motorcycles": 697,
        "bicycles": 111
    }, {
        "dte": "2021-04-12",
        "cars": 1213,
        "motorcycles": 633,
        "bicycles": 87
    }, {
        "dte": "2021-04-13",
        "cars": 1199,
        "motorcycles": 621,
        "bicycles": 79
    }, {
        "dte": "2021-04-14",
        "cars": 1110,
        "motorcycles": 210,
        "bicycles": 81
    }, {
        "dte": "2021-04-15",
        "cars": 1165,
        "motorcycles": 232,
        "bicycles": 75
    }, {
        "dte": "2021-04-16",
        "cars": 1145,
        "motorcycles": 219,
        "bicycles": 88
    }, {
        "dte": "2021-04-17",
        "cars": 1163,
        "motorcycles": 201,
        "bicycles": 82
    }, {
        "dte": "2021-04-18",
        "cars": 1180,
        "motorcycles": 285,
        "bicycles": 87
    }, {
        "dte": "2021-04-19",
        "cars": 1159,
        "motorcycles": 277,
        "bicycles": 71
    }];


    // Create axes
    // https://www.amcharts.com/docs/v5/charts/xy-chart/axes/
    var xAxis = chart.xAxes.push(am5xy.DateAxis.new(root, {
        baseInterval: { timeUnit: "day", count: 1 },
        //startLocation: 0.5,
        //endLocation: 0.5,
        renderer: am5xy.AxisRendererX.new(root, {}),
        //tooltip: am5.Tooltip.new(root, {})
    }));

    //xAxis.data.setAll(data);

    var yAxis = chart.yAxes.push(am5xy.ValueAxis.new(root, {
        //min: 0,
        //max: 3000,
        calculateTotals: true,
        renderer: am5xy.AxisRendererY.new(root, {})
    }));

    // Add series
    // https://www.amcharts.com/docs/v5/charts/xy-chart/series/

    function createSeries(name, field) {
        var series = chart.series.push(am5xy.LineSeries.new(root, {
            stacked: true,
            name: name,
            xAxis: xAxis,
            yAxis: yAxis,
            valueYField: field,
            valueXField: "dte",
            tooltip: am5.Tooltip.new(root, {
                pointerOrientation: "horizontal",
                labelText: "[bold]{name}[/]\n{categoryX}: {valueY}"
            })
        }));

        series.fills.template.setAll({
            fillOpacity: 0.5,
            visible: true
        });

        // Set up data processor to parse string dates
        // https://www.amcharts.com/docs/v5/concepts/data/#Pre_processing_data
        series.data.processor = am5.DataProcessor.new(root, {
            dateFormat: "yyyy-MM-dd",
            dateFields: ["dte"]
        });

        series.data.setAll(data);
        series.appear(1000);
    }

    createSeries("Cars", "cars");
    createSeries("Motorcycles", "motorcycles");
    createSeries("Bicycles", "bicycles");

    // Add scrollbar
    // https://www.amcharts.com/docs/v5/charts/xy-chart/scrollbars/
    chart.set("scrollbarX", am5.Scrollbar.new(root, {
        orientation: "horizontal"
    }));

    // Create axis ranges
    // https://www.amcharts.com/docs/v5/charts/xy-chart/axes/axis-ranges/
    var rangeDataItem = xAxis.makeDataItem({
        category: "2001",
        endCategory: "2003"
    });

    var range = xAxis.createAxisRange(rangeDataItem);

    rangeDataItem.get("grid").setAll({
        stroke: am5.color(0x00ff33),
        strokeOpacity: 0.5,
        strokeDasharray: [3]
    });

    rangeDataItem.get("axisFill").setAll({
        fill: am5.color(0x00ff33),
        fillOpacity: 0.1
    });

    rangeDataItem.get("label").setAll({
        inside: true,
        text: "Fines for speeding increased",
        rotation: 90,
        centerX: am5.p100,
        centerY: am5.p100,
        location: 0,
        paddingBottom: 10,
        paddingRight: 15
    });


    var rangeDataItem2 = xAxis.makeDataItem({
        category: "2007"
    });

    var range2 = xAxis.createAxisRange(rangeDataItem2);

    rangeDataItem2.get("grid").setAll({
        stroke: am5.color(0x00ff33),
        strokeOpacity: 1,
        strokeDasharray: [3]
    });

    rangeDataItem2.get("label").setAll({
        inside: true,
        text: "Motorcycle fee introduced",
        rotation: 90,
        centerX: am5.p100,
        centerY: am5.p100,
        location: 0,
        paddingBottom: 10,
        paddingRight: 15
    });

    // Make stuff animate on load
    // https://www.amcharts.com/docs/v5/concepts/animations/
    chart.appear(1000, 100);

    //}); // end am5.ready()
</script>
```

Did i miss something ?

Thanks,

Fred

**Comment by martynasma** [comment]
Fixed in [5.0.7](https://github.com/amcharts/amcharts5/releases/tag/5.0.7).

## [5.0.7] - 2021-11-19

### Added
- New setting `pan` (default: `"none"`) added to `AxisRendererX` and `AxisRendererY`. If set to `"zoom"` will enable zoom of axis scale by dragging in axis label area.

### Changed
- Heat rules will not longer apply setting value if target data item does not have any numeric value.
- Hovering legend item will now simulate hover on related item on some charts (e.g. `PieSeries`, `Hierarchy`, `Venn`).

###
- `HeatLegend` was acting out when its `showValue()` method was called with non-numeric parameter.
- If series was pre-hidden before chart init, its status was not being correctly reflected in related legend item.
- Locales in script version were not working properly.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by bereldhuin** [comment]
Hi,

Thanks for the fix, it works fine now :-)
